source/rover.js

```diff
@@ -17,6 +17,15 @@ this.initialize = function() {
     if ( this.id === "source/rover.vwf" ) {
         return;
     }
+
+    //TODO: Set this properly (we shouldn't have to set this explicity in here...
+    // this.transform gets set automatically for ONE of the rovers, but not the other one)
+    this.transform = [
+        1,  0,  0,  0,
+        0,  1,  0,  0,
+        0,  0,  1,  0,
+        0,  0,  0,  1
+    ];
```

Comment:
If I comment this out, I run into trouble in the `placeOnTerrain` function - in particular, around line 108 (where `this.transform` is accessed). Currently, the first rover listed in `index.vwf.yaml` gets `this.transform` assigned before `placeOnTerrain` is called. The next two rovers have an undefined `this.transform`.

I.e. if the rovers are listed in the order `rover`,  `rover2`, `rover3` in `index.vwf.yaml`, then `rover` has `this.transform` assigned before `placeOnTerrain` is called, but `rover3` and `rover2` have an undefined `this.transform`. If we rerarrange `index.vwf.yaml` such that the rovers are ordered as `rover2`, `rover`, `rover3`, then `rover2` gets a transform before placeOnTerrain is called, but `rover` and `rover3` have undefined transforms.